frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx

```diff
@@ -40,10 +40,6 @@ export function InsightsTable({ isLegend = true, showTotalCount = false }: Insig
     const { calcColumnState } = useValues(logic)
     const { setCalcColumnState } = useActions(logic)
 
-    if (indexedResults.length === 0 || !indexedResults?.[0]?.data) {
```

Comment:
I'm not sure if we should remove this, IIRC @alexkim205 introduced the second check because of a Sentry error that came popping up with an edge case.

Reply:
What's the edge case? 

Removing this didn't raise any type errors so likely to cause further issues down the line.

I'd rather not leave code we're not sure of in :)

Reply:
Yup just checked L124, and looks like we have a check there